Expensify/App
# Added support for dynamic port

Please review @roryabraham @marcaaron 

### Details
Added a script under the desktop folder named `start.js` which runs the same command as the npm script to build the desktop dev server but we first find an available port which is then used to run the Renderer process.

### Fixed Issues
<Please replace GH_LINK with the link to the GitHub issue this Pull Request is fixing>
Fixes #1571 

### Tests
 1. Run the webserver using `npm run web`. Wait for it to successfully start and then note the Port used.
 2. Run the Desktop server using `npm run desktop` which should use a different port than the port used on the web process.
 3. To test the regressions, run various servers to block the Ports and then run the desktop server. It should start with an available port.

### Tested On


### Screenshots
N/A
